ssh/tailssh/incubator.go

```diff
@@ -268,53 +245,185 @@ func beIncubator(args []string) error {
 		defer sessionCloser()
 	}
 
-	var groupIDs []int
-	for _, g := range strings.Split(ia.groups, ",") {
-		gid, err := strconv.ParseInt(g, 10, 32)
-		if err != nil {
+	if attemptLoginShell {
+		// We weren't able to use login, maybe we can use su.
```

Comment:
```suggestion
		// If we got here, we weren't able to use login (because tryExecLogin returned without replacing the running process), maybe we can use su.
```